[DOC] Update active record basic guide for create method (#52315)
rails/rails: guides/source/active_record_basics.md

more.

Methods like `save`, `create` and `update` validate a model before persisting it
to the database. If the model is invalid, no database operations are performed. In
this case the `save` and `update` methods return `false`. The `create` method still

> returns the ActiveRecord object, which can be checked for errors. All of these

Reviewer: ```suggestion
returns the object, which can be checked for errors. All of these
```

more.

Methods like `save`, `create` and `update` validate a model before persisting it

> to the database. If the model is invalid, the `save` and `update` methods return `false`, and no database operations are performed. However, the `create` method does not return `false`; instead, it returns the ActiveRecord object, which can be checked for errors.  All of these methods have a bang counterpart

Reviewer 2: can you also split the sentence into multiple lines. please check character count of other sentences in each line.

Reviewer 3: It might be unclear that "no database operations are performed" for `create` as well. So maybe separate that?
Also I think it's more clear to just explain what `create` returns instead of what it doesn't return.
```suggestion
to the database. If the model is invalid, no database operations are performed. In this case the `save` and `update` methods return `false` instead of `true`. The `create` method still returns the ActiveRecord object, which can be checked for errors. All of these methods have a bang counterpart
```

Author: Sounds good to me.

Author: Done.

more.

Methods like `save`, `create` and `update` validate a model before persisting it

> to the database. When a model is invalid `save` and `update` methods return `false` and no
> database operations are performed. The create method, however, does not return `false`; it will return the ActiveRecord object, which can be checked for errors. All of these methods have a bang counterpart

Reviewer 2: ```suggestion
to the database. If the model is invalid, the `save` and `update` methods return `false`, and no database operations are performed. However, the `create` method does not return `false`; instead, it returns the ActiveRecord object, which can be checked for errors.  All of these methods have a bang counterpart
```

ActiveRecord::RecordInvalid: Validation failed: Name can't be blank
```

> In the example below, the create method returns an ActiveRecord object, even if it is invalid, which can then be checked for errors.

Reviewer 2: ```suggestion
The `create` method in the example returns an ActiveRecord object, regardless of its validity. You can then inspect this object for any errors.
```

ActiveRecord::RecordInvalid: Validation failed: Name can't be blank
```

> The `create` method always returns an ActiveRecord object, regardless of

Reviewer: ```suggestion
The `create` method always returns an object, regardless of
```

Reviewer 3: In Ruby everything is a object 😄, so this might be confusing (it won't return an instance of the `FalseClass`).
Maybe:
```suggestion
The `create` method always returns the record, regardless of
```

Author: `record` sounds more generic. I'm changing it to the `the record`

Author: @p8 Changes are done. Please review.

> The `create` method in the example returns an ActiveRecord object, regardless of

Reviewer 3: I think we can be a bit stricter here, `create` always returns an ActiveRecord object:
```suggestion
The `create` method always returns an ActiveRecord object, regardless of
```

Author: @p8 
sure, done.
